pyproject.toml

```diff
@@ -209,4 +208,4 @@ addopts = "--color=yes --cov-fail-under=90 --cov=cpplint"
 python_files = [ "*test.py" ]
 required_plugins = [ "pytest-cov", "pytest-timeout" ]
 testpaths = [ "." ]
-timeout = 60
+timeout = 481
```

Comment:
This seems like a massive change.  Is there a particular versions of Python and/or operating system that is so slow?

Reply:
It's not that it's slow, it's that I oftentimes find myself having to debug the tests, for which 1 minute is not enough for. My average debugging time is about 5 minutes, and 418 is my favorite number.

Reply:
481 != 418 ;-)